core/src/main/java/io/questdb/cairo/pool/ReaderPool.java

```diff
@@ -82,6 +108,8 @@ public static class R extends TableReader implements PoolTenant<R> {
         private final int index;
         private final ReaderListener readerListener;
         private boolean detached;
+        // May be used to track usage of detached readers.
+        private int detachedRefCount;
```

Comment:
There is a suspicion that readers can be closed in a different thread from the MV refresh job thread. 
Here is possibly the issue about that https://github.com/questdb/questdb/issues/5425

I think it's still ok to use this without syncronisations/atomic, if the value is read stale 1 instead of 0 there will be an additional reader copy, but please consider this/add a comment that it's ok.

Reply:
Added a clarifying comment in e65c5facca8c7565b190daf11b2ca8d1fef51ff3